x/oracle/client/cli/query_native_token.go

```diff
@@ -0,0 +1,100 @@
+package cli
+
+import (
+	"github.com/ExocoreNetwork/exocore/x/oracle/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+func CmdQueryStakerInfos() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-staker-infos [assetID]",
+		Short: "shows all staker infos including stakerAddr, validators of that staker, latest balance...",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			assetID := args[0]
```

Comment:
Stateless validate this via `assetstypes.ValidateID` ? Not a blocker.

Reply:
updated